poetry/core/json/schemas/poetry-schema.json

```diff
@@ -479,7 +479,7 @@
         },
         "multiple-constraints-dependency": {
             "type": "array",
-            "minItems": 1,
+            "minItems": 2,
```

Comment:
Should we enforce this? It might break some existing projects. And it is a nice to have to be able to specify an array of 1 :)

Reply:
Thanks fro pointing this out :-) I changed it back to `1`.